projects/angular/src/data/datagrid/built-in/filters/datagrid-numeric-filter.ts

```diff
@@ -53,6 +62,8 @@ export class DatagridNumericFilter<T = any>
 {
   @Input('clrFilterMinPlaceholder') minPlaceholder: string;
   @Input('clrFilterMaxPlaceholder') maxPlaceholder: string;
+  @Input('clrFilterMinValue') minLabel: string;
+  @Input('clrFilterMaxValue') maxValue: string;
```

Comment:
```suggestion
  @Input('clrFilterFromValue') fromLabel: string;
  @Input('clrFilterToValue') toValue: string;
```

I think I made a bad suggestion before... This mixes min/from and max/to in the label naming...

Reply:
I am really sorry. I should have paid more attention when reviewing.